web/models/class/class.ex

```diff
@@ -82,4 +85,27 @@ defmodule CoursePlanner.Class do
   end
   def validate_offered_course(changeset), do: changeset
 
+  defp validate_date(%{valid?: true, changes: %{date: date, offered_course_id: oc_id}} = changeset) do
```

Comment:
is this working properly with class updates? it seems to me there wouldn't be a change on the field `offered_course_id` in an update